app/views/admin/shared/_booth_search.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_for(Poll::Booth.new, url: url, as: :booth, method: :get) do |f| %>
+  <div class="small-12 medium-6">
+    <div class="input-group">
+      <%= text_field_tag :search, "", placeholder: t("admin.shared.booths_search.placeholder") %>
```

Comment:
The form is probably more usable if we maintain the search term in the input, for example with:
```erb
      <%= text_field_tag :search, params[:search], placeholder: t("admin.shared.booths_search.placeholder") %>
```
I know we use an empty string in other similar forms. Our bad :sweat_smile:. Sorry for that! :pray: